spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -818,18 +818,42 @@
     end
   end
 
+  describe "#calculate_line_15" do
+    before do
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_9).and_return 2
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_10a).and_return 4
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_11).and_return 6
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_13).and_return 8
+    end
+    it "sums lines 8 - 14" do
+      instance.calculate
+      expect(instance.lines[:MD502_LINE_15].value).to eq 20
+    end
+  end
+
+  describe "#calculate_line_16" do
+    before do
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_7).and_return 150
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_15).and_return 50
+    end
+    it "subtracts line 15 from line 7" do
+      instance.calculate
+      expect(instance.lines[:MD502_LINE_16].value).to eq 100
+    end
+  end
+
   describe "#calculate_line_17" do
     context "when method is standard" do
       [
         [["single", "married_filing_separately", "dependent"], [
           [12_000, 1_800],
           [17_999, 17_999 * 0.15],
-          [18_000, 2_700],
+          [18_001, 2_700],
```

Comment:
changed the tests to expose the above mentioned [bug](https://github.com/codeforamerica/vita-min/pull/4980/files#r1842913316).